.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.idea/
```

Comment:
In PyCharm, these files are added automatically and I think it is worth adding them to gitignore.